src/Our.Umbraco.Ditto/ComponentModel/ConversionHandlers/DittoConversionHandlerRegistry.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Our.Umbraco.Ditto
+{
+    /// <summary>
+    /// Registory for globaly registered conversion handlers.
+    /// </summary>
+    internal class DittoConversionHandlerRegistry
+    {
+        /// <summary>
+        /// The cache for storing handler information.
+        /// </summary>
+        private static readonly Dictionary<Type, IList<Type>> HandlerCache
```

Comment:
I can't because it's a list in a dictionary. With ConcurrentDictionary, access to the dictionary would be thread safe, but access to the list in the dictionary wouldn't, so I use a lock instead.